### PR TITLE
feat: workflow: update reference data & remote config

### PR DESCRIPTION
**Pull request by torfjor**
Syncs src/reference-data/*.json every hour and opens a PR if there are any diffs.

On push: uses an external tool: github.com/atb-as/remoteconfig/cmd/rc that lets you publish key=value pairs as default values in remote config. Need to revisit later when we need to differentiate on environments.

### CODE REVIEW

**Comment by mikaelbr** [comment]
@gorandalum or @cbrevik Could any of you who know the ref data/remote config knowledge look into reviewing this?

File: .github/workflows/publish_remote_config.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Publish remote config
+
+on:
+  push:
+    branches:
+      - master
```
Comment (cbrevik) [inline .github/workflows/publish_remote_config.yml:6]: Maybe add a path here so it only runs if `reference-data` is updated?

Haven't tested it out, but something like:

```suggestion
  push:
    branches:
      - master
    paths:
      - 'src/reference-data/*.json'
```

**Review by cbrevik — APPROVED** [review]
LGTM, just a suggested change

File: .github/workflows/publish_remote_config.yml (hunk 1 above)
Comment (torfjor) [reply .github/workflows/publish_remote_config.yml:6]: Looks like the workflow docs use double star for wildcards. Will do some local testing.

**Pull request merged**